spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -120,6 +120,11 @@ def _setup_matplotlib(self):
     # If we don't have matplotlib installed don't bother continuing
     try:
       import matplotlib
+
+      #matplotlib.rcParams['savefig.format'] is only avail in version 1.2.0 and above
+      from distutils.version import LooseVersion
+      if LooseVersion("1.2.0") >= LooseVersion(matplotlib.__version__):
+        return
```

Comment:
Sure, done. Also, have reverted the changes from `zeppelin_pyspark.py`. Let me know your thoughts.